src/app/about/about.component.ts

```diff
@@ -0,0 +1,29 @@
+import {Component, OnInit} from '@angular/core';
+import {ContactService} from '../services/contact.service';
+import {SmsinfoService} from '../services/smsinfo.service';
+
+@Component({
+  selector: 'app-about',
+  templateUrl: './about.component.html',
+  styleUrls: ['./about.component.scss']
+})
+export class AboutComponent implements OnInit {
+  phoneNumber = '';
+
+  constructor(private contactService: ContactService,
+              private smsInfoService: SmsinfoService) {}
+
+  getContactEmail() {
+    return this.contactService.email;
+  }
+
+  textPhoneNumber() {
+    return this.phoneNumber;
+  }
+
+  ngOnInit() {
+    this.smsInfoService.getPhoneNumber().subscribe((phoneNum) => {
```

Comment:
First, when you subscribe, you need to unsubscribe. BUT you never have to directly subscribe if you use the `async` pipe in your angular template.

Instead of subscribing set a public local variable: `phoneNumber$: Observable<string>;` and have this method set the  observable from the service call and then can get the data from `phoneNumber$ | async as phoneNumber`